src/main/views/macro/address.njk

```diff
@@ -9,7 +9,7 @@
     </form>
 {% endmacro %}
 
-{% macro postcodeLookupAddressFragment(form, name, postcodeLabel) %}
+{% macro postcodeLookupAddressFragment(form, name, postcodeLabel, ariaText ='') %}
```

Comment:
Please make use of this pull request. https://github.com/hmcts/cmc-citizen-frontend/pull/2458
I have added label to each input and you can use span with invisible label and remove the aria-label.